src/runtimes/docker/utils/getNativeArchitecture.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as os from 'os';
+
+export type CpuArchitecture =
+    | 'amd64'
+    | '386'
+    | 'arm64'
+    | 'arm'
+    | 'mips'
+    | 'mipsle'
+    | 'ppc64'
+    | 's390x';
```

Comment:
To cover for any unknown cases, we can say `| string` to allow any value
```suggestion
    | 's390x'
    | string;
```